src/preferences/dialog/dlgprefsound.cpp

```diff
@@ -18,6 +18,8 @@
 namespace {
 
 const QString kAppGroup = QStringLiteral("[App]");
+const QString kMicMonitorHintTrString =
+        QObject::tr("Refer to the Mixxx User Manual for details.");
```

Comment:
The translation has moved form the "DlgPrefSound" to the "QObject" context. 
```suggestion
        DlgPrefSound::tr("Refer to the Mixxx User Manual for details.");
```
would work, but it will probably fail with linters. 

There is another issue that we don't know when the variable is initialized. Most likely just before usage, so it will work. however there is no guarantee that it happens after reading the selected language.  

So it is probably best to move this constant to: 
`DlgPrefSound::checkLatencyCompensation()`